assets/evm/v1/assets_dev.json

```diff
@@ -11,5 +11,445 @@
                 "contractAddress": "0xeFAeeE334F0Fd1712f9a8cc375f427D9Cdd40d73"
             }
         ]
+    },
+    {
+        "symbol": "USDC",
+        "precision": 6,
+        "name": "USD Coin",
+        "priceId": "usd-coin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/USDC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x818ec0A7Fe18Ff94269904fCED6AE3DaE6d6dC0b"
+            }
+        ]
+    },
+    {
+        "symbol": "USDC",
+        "precision": 6,
+        "name": "USD Coin",
+        "priceId": "usd-coin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/USDC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x931715FEE2d06333043d11F658C8CE934aC61D0c"
+            }
+        ]
+    },
```

Comment:
Lets leave only one USDC for now (one that is "multichain")

Reply:
left multchain USDC